web/app/fr/gospeak/web/pages/orga/partials/contactItem.scala.html

```diff
@@ -0,0 +1,12 @@
+@import fr.gospeak.core.domain.{Group, Proposal, User, Event}
+@import fr.gospeak.web.pages.partials.display.html._
+@import fr.gospeak.web.utils.Formats
+@import fr.gospeak.web.pages.orga.partners.contacts.routes.ContactCtrl
+
+@import fr.gospeak.core.domain.Contact
+@import fr.gospeak.core.domain.Partner
+
+@(group: Group, partner: Partner, contact: Contact)
+<a class="list-group-item list-group-item-action" href="@ContactCtrl.detail(group.slug, partner.slug, contact.id)">
+    <h5 class="mb-1">@contact.firstName.value</h5>
```

Comment:
can be nice to have the full name here ;)

Reply:
Done